src/worker/config.rs

```diff
@@ -23,10 +24,13 @@ pub struct WorkerConfig {
     /// "outstanding" until all activations it requires have been completed.
     #[builder(default = "100")]
     pub max_outstanding_workflow_tasks: usize,
-    /// The maximum allowed number of activity tasks that will ever be given to this worker at one
-    /// time.
+    /// The maximum number of activity tasks that will ever be given to this worker concurrently
     #[builder(default = "100")]
     pub max_outstanding_activities: usize,
+    /// The maximum number of local activity tasks that will ever be given to this worker
+    /// concurrently
+    #[builder(default = "100")]
```

Comment:
These are 1k default in Go for all 3 of these

Reply:
Remember this is just the default for core. Lang impl is free to select it's own "defaults" that make sense.